Formula/ford.rb

```diff
@@ -116,8 +116,8 @@ def install
       You can have as many paragraphs as you like here and can use headlines, links,
       images, etc. Basically, you can use anything in Markdown and Markdown-Extra.
       Furthermore, you can insert LaTeX into your documentation. So, for example,
-      you can provide inline math using like \( y = x^2 \) or math on its own line
-      like \[ x = \sqrt{y} \] or $$ e = mc^2. $$ You can even use LaTeX environments!
+      you can provide inline math using like ( y = x^2 ) or math on its own line
+      like [ x = \sqrt{y} ] or $$ e = mc^2. $$ You can even use LaTeX environments!
```

Comment:
```suggestion
      like \\[ x = \sqrt{y} \\] or $$ e = mc^2. $$ You can even use LaTeX environments!
```
More generally, I think most/all the backslashes in this test need escaping.